libflux/flux/src/lib.rs

```diff
@@ -137,7 +139,7 @@ pub unsafe extern "C" fn flux_ast_get_error(
     let mut errs = ast::check::check(ast_pkg);
     if !errs.is_empty() {
         let err = Vec::remove(&mut errs, 0);
-        Some(Box::new(ErrorHandle { err: Box::new(err) }))
+        Some(Box::from(err))
```

Comment:
I think `Some(err.into())` is more clear. I think I do that in other locations too.

Same in all the below places too.